### PR TITLE
[releaser] Several fixes of releaser

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| During release of 10.0.5 I have found several problems, that are fixed in this PR.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: utils/releaser/src/ReleaseWorker/StageWorkerInterface.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string;
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
+     */
+    public function work(Version $version, string $initialBranchName = 'master'): void;
```
Comment (RostislavKreisinger) [inline utils/releaser/src/ReleaseWorker/StageWorkerInterface.php:26]: initialBrancheName is required, so default value may not be here ?

File: utils/releaser/src/ReleaseWorker/AfterRelease/RemoveLockFilesReleaseWorker.php [hunk 2]
```diff
@@ -21,22 +21,23 @@ public function getDescription(Version $version): string
...
-    public function work(Version $version): void
+    public function work(Version $version, string $initialBranchName = 'master'): void
     {
         $this->processRunner->run('git rm project-base/composer.lock');
         $this->processRunner->run('git rm project-base/package-lock.json');
         $this->processRunner->run('git rm project-base/migrations-lock.yml');
         $this->commit('removed locked versions of dependencies for unreleased version');
 
-        if ($this->initialBranchName === 'master') {
+        if ($this->currentBranchName === 'master') {
```
Comment (RostislavKreisinger) [inline utils/releaser/src/ReleaseWorker/AfterRelease/RemoveLockFilesReleaseWorker.php:35]: master could be a constant

File: utils/releaser/src/ReleaseWorker/AbstractVerifyInitialBranchReleaseWorker.php [hunk 3]
```diff
@@ -10,9 +10,10 @@ abstract class AbstractVerifyInitialBranchReleaseWorker extends AbstractShopsysR
 {
     /**
      * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
      * @return string
      */
-    public function getDescription(Version $version): string
+    public function getDescription(Version $version, string $initialBranchName = 'master'): string
```
Comment (RostislavKreisinger) [inline utils/releaser/src/ReleaseWorker/AbstractVerifyInitialBranchReleaseWorker.php:16]: The default value should not be needed here. This is in other classes to.

File: utils/releaser/src/ReleaseWorker/StageWorkerInterface.php (hunk 1 above)
Comment (TomasLudvik) [reply utils/releaser/src/ReleaseWorker/StageWorkerInterface.php:26]: Sadly interface says, that this method has only one argument, so in order to add another argument, it has to have default value set

File: utils/releaser/src/ReleaseWorker/AbstractVerifyInitialBranchReleaseWorker.php (hunk 3 above)
Comment (TomasLudvik) [reply utils/releaser/src/ReleaseWorker/AbstractVerifyInitialBranchReleaseWorker.php:16]: Same as previous answer

File: utils/releaser/src/ReleaseWorker/AfterRelease/RemoveLockFilesReleaseWorker.php (hunk 2 above)
Comment (TomasLudvik) [reply utils/releaser/src/ReleaseWorker/AfterRelease/RemoveLockFilesReleaseWorker.php]: I have added new constant

File: utils/releaser/src/ReleaseWorker/StageWorkerInterface.php (hunk 1 above)
Comment (RostislavKreisinger) [reply utils/releaser/src/ReleaseWorker/StageWorkerInterface.php]: oki, in first view looks like new interface instead of ReleaseWorkerInterface

**Review by RostislavKreisinger — APPROVED** [review]

**Pull request merged**